experiments/functional/zfs-LocalPV/zfspv-custom-topology/app_gen_I.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+  
+set -e
+
+mkdir app_yamls_I
+
+for i in $(seq 1 5)
+do
+        sed "s/pvc-custom-topology/pvc-custom-topology$i/g" busybox_I.yml > app_yamls_I/busybox$i.yml
```

Comment:
can you add `-` before appending for better readability

Reply:
done